Validators.js

```diff
@@ -134,28 +134,30 @@ function areEventParametersValid(functionName, eventName, segmentation, eventCou
         return false;
     }
 
-    // validate segmentation values
     if (segmentation) {
         for (const key in segmentation) {
             const value = segmentation[key];
             const valueType = typeof value;
-            if (value && valueType !== "string" && valueType !== "number" && valueType !== "boolean") {
-                L.w(`${functionName}, segmentation value: [${value}] for the key: [${key}] must be a number, string or boolean!`);
+            if (Array.isArray(value)) {
```

Comment:
let type checking inside the array to underlying sdks. they will omit those values and send the event to server (they will not cancel the event recording)